package.json

```diff
@@ -884,20 +885,22 @@
     "@types/sinon": "^9.0.1",
     "@types/vscode": "^1.49.0",
     "@types/ws": "^7.2.4",
-    "@typescript-eslint/eslint-plugin": "^2.19.2",
-    "@typescript-eslint/parser": "^2.19.2",
+    "@typescript-eslint/eslint-plugin": "^4.11.1",
```

Comment:
This and "@typescript-eslint/parser" are already 4.12.0

Reply:
👍  will rebase once the editing playground results pr goes through and then we can look at this again and merge.